Sort plot-schema and add test to track plot-schema changes (#5776)
plotly/plotly.js: CONTRIBUTING.md

There is also a search bar in the top right of the dashboard. This fuzzy-searches
image mocks based on their file name and trace type.

#### Step 5: Regenerate plot-schema in "test" folder then review & commit potential changes

```bash
npm run schema
```

#### Step 6: Review & commit potential changes made to test/plot-schema.json

> > If you are editing attribute descriptions or implementing a new feature this file located in test folder would record the proposed changes to the API. This test file is different from the other plot-schema.json file located in the dist folder and should only be updated by maintainers at the release time.

Reviewer: ```suggestion
> If you are editing attribute descriptions or implementing a new feature this file located in the test folder records the proposed changes to the API. Note that there is another plot-schema.json file located in the dist folder, which should only be updated by the maintainers at release time.
```
